Source/Scene/Cesium3DTileset.js

```diff
@@ -1930,6 +1947,9 @@ define([
         // Destroy debug labels
         this._tileDebugLabels = this._tileDebugLabels && this._tileDebugLabels.destroy();
 
+        // Destroy clipping plane collection
```

Comment:
This comment and `// Destroy debug labels` above are not needed.

Just like `++i; // add one to i` would not be needed.